www/pages/plan-trip/plan-trip-controller.js

```diff
@@ -320,6 +320,10 @@ angular.module('pvta.controllers').controller('PlanTripController', function ($s
     });
   };
 
+  var checkRouteTransit = function(route) {
```

Comment:
Could it be renamed?  If I didn't know what it did, I wouldn't know what "check route transit" means.